### ISSUE TITLE
CultureNotFoundException in SchedulingExtensions.GenerateResourceScrapingJobName

### ISSUE THREAD

**Issue by tomkerkhove** [tomkerkhove/promitor#2089]
### Report

Exception happens during startup on https://dev.azure.com/tomkerkhove/Promitor/_build/results?buildId=8562&view=logs&j=30190cae-7ed8-53e7-ee11-9246d6a79584&t=f881dc73-f627-54ba-40b1-e5e0d0a590fa&l=33
```
fail: Promitor.Agents.Scraper.Startup[0]
      Failure to create job name for scraping resources for list of metrics which includes availabilityResults/availabilityPercentage.
      System.Globalization.CultureNotFoundException: Only the invariant culture is supported in globalization-invariant mode. See https://aka.ms/GlobalizationInvariantMode for more information. (Parameter 'name')
      en-US is an invalid culture identifier.
         at System.Globalization.CultureInfo..ctor(String name, Boolean useUserOverride)
         at System.Globalization.CultureInfo..ctor(String name)
         at CronExpressionDescriptor.ExpressionParser..ctor(String expression, Options options)
         at CronExpressionDescriptor.ExpressionDescriptor.GetDescription(DescriptionTypeEnum type)
         at CronExpressionDescriptor.ExpressionDescriptor.GetDescription(String expression, Options options)
         at CronExpressionDescriptor.ExpressionDescriptor.GetDescription(String expression)
         at Microsoft.Extensions.DependencyInjection.SchedulingExtensions.GenerateResourceScrapingJobName(MetricsDeclaration metricsDeclaration, ILogger`1 logger) in /src/Promitor.Agents.Scraper/SchedulingExtensions.cs:line 133
[19:29:02 INF] Starting validation of Promitor setup
```

### Expected Behavior

All metrics are being scraped.

### Actual Behavior

```
fail: Promitor.Agents.Scraper.Startup[0]
      Failure to create job name for scraping resources for list of metrics which includes availabilityResults/availabilityPercentage.
      System.Globalization.CultureNotFoundException: Only the invariant culture is supported in globalization-invariant mode. See https://aka.ms/GlobalizationInvariantMode for more information. (Parameter 'name')
      en-US is an invalid culture identifier.
         at System.Globalization.CultureInfo..ctor(String name, Boolean useUserOverride)
         at System.Globalization.CultureInfo..ctor(String name)
         at CronExpressionDescriptor.ExpressionParser..ctor(String expression, Options options)
         at CronExpressionDescriptor.ExpressionDescriptor.GetDescription(DescriptionTypeEnum type)
         at CronExpressionDescriptor.ExpressionDescriptor.GetDescription(String expression, Options options)
         at CronExpressionDescriptor.ExpressionDescriptor.GetDescription(String expression)
         at Microsoft.Extensions.DependencyInjection.SchedulingExtensions.GenerateResourceScrapingJobName(MetricsDeclaration metricsDeclaration, ILogger`1 logger) in /src/Promitor.Agents.Scraper/SchedulingExtensions.cs:line 133
[19:29:02 INF] Starting validation of Promitor setup
```

### Steps to Reproduce the Problem

Run CI

### Component

Scraper

### Version

master-branch

### Configuration

Configuration:

```yaml
# Add your scraping configuration here
```

### Logs

```shell
example
```

**Comment by notanops** [comment]
Hi,

It appears that the metric `allusedmemorypercentage` also triggers `System.Globalization.CultureNotFoundException`

Metric Definition: 

```yaml
  - name: azure_redis_cache_memory_percentage
    description: "The percentage of cache memory used for key/value pairs."
    resourceType: RedisCache
    scraping:
      schedule: "0 */2 * ? * *"
    azureMetricConfiguration:
      metricName: allusedmemorypercentage
      aggregation:
        type: Maximum
        interval: 00:01:00
```

Logs: 

```
fail: Promitor.Agents.Scraper.Startup[0]
      Failure to create job name for scraping resources for list of metrics which includes allusedmemorypercentage.
      System.Globalization.CultureNotFoundException: Only the invariant culture is supported in globalization-invariant mode. See https://aka.ms/GlobalizationInvariantMode for more information. (Parameter 'name')
      en-US is an invalid culture identifier.
         at System.Globalization.CultureInfo..ctor(String name, Boolean useUserOverride)
         at System.Globalization.CultureInfo..ctor(String name)
         at CronExpressionDescriptor.ExpressionParser..ctor(String expression, Options options)
         at CronExpressionDescriptor.ExpressionDescriptor.GetDescription(DescriptionTypeEnum type)
         at CronExpressionDescriptor.ExpressionDescriptor.GetDescription(String expression, Options options)
         at CronExpressionDescriptor.ExpressionDescriptor.GetDescription(String expression)
         at Microsoft.Extensions.DependencyInjection.SchedulingExtensions.GenerateResourceScrapingJobName(MetricsDeclaration metricsDeclaration, ILogger`1 logger) in /src/Promitor.Agents.Scraper/SchedulingExtensions.cs:line 133
[13:16:29 INF] Starting validation of Promitor setup
```
After some tests, none of the metrics are working (count, total, maximum, percentage or not), they all raise the same exception for Azure Redis Cache.

Hoping this helps you.

**Comment by tomkerkhove** [comment]
🤦 

I'll pick this up asap

**Comment by tomkerkhove** [comment]
This is available in Scraper v2.8.0 - https://github.com/tomkerkhove/promitor/releases/tag/Scraper-v2.8.0
